### ISSUE TITLE
Support for IfElse, and Cond

### ISSUE THREAD

**Issue by EfstathiadisD** [remeda/remeda#214]
One of the most beautiful helpers in Ramada are 'Cond' and 'IfElse'.

Is there any chance we can bring them into remeda? Basically the only thing that bugs me. But I understand it's early days.

Happy to Submit a PR if you provide some instructions.. Thanks for the hard work. 😊

**Comment by TkDodo** [comment]
what would you need from us to make a PR ?

**Comment by thelawclaw11** [comment]
This is something I'd like to see as well. I'll work on getting a PR ready!

**Comment by cjquines** [comment]
i beleive this is closed by #331 --- i think `conditional` works fine for `ifElse`:

```ts
R.conditional([
  [R.isTruthy, trueCase],
  [R.not(R.isTruthy), falseCase],
]);
```
